lib/select2-3.5.1/select2.css

```diff
@@ -558,6 +536,14 @@ html[dir="rtl"] .select2-container-multi .select2-choices li
     color: #999 !important;
 }
 
+.select2-search input.select2-input {
+    background: #EBEBEB;
+    background-image: none !important;
+    border: 0;
+    padding: 10px 15px;
+    line-height: 0;
+}
+
 .select2-container-multi .select2-choices .select2-search-choice {
     padding: 3px 5px 3px 18px;
     margin: 3px 0 3px 5px;
```

Comment:
@reizelfranz Sorry I didn't notice this change before. We've never changed css files inside the lib folder, as those are raw downloaded library files.  Changing these here means that we won't be able to upgrade these libraries without difficulty in the future.  Can you define these overrides in our own .less files?  Here are some examples:

https://github.com/bcjobs/assets/blob/master/css/_jobcentre_theme.less#L944

It looks in those examples, someone added the `body` prefix to increase specificity, but even with the same specificity, our rules should win because our css files are loaded after select2.

If you agree with this approach, I can show you how easily you can revert these changes to this files (after they've been ported) using git.